codegen/Templates/python/api_client.mustache

```diff
@@ -159,12 +159,11 @@ class ApiClient(object):
         # request url
         url = ''
         if six.PY3:
-            url = self.configuration.host + '/' + \
-                  self.configuration.api_version + resource_path
+            url = self.configuration.host + self.configuration.api_version + \
+                  resource_path
         else:
-            url = (self.configuration.host + '/' +
-                   self.configuration.api_version + resource_path)\
-                .encode('utf8')
+            url = (self.configuration.host + self.configuration.api_version +
+                   resource_path).encode('utf8')
 
```

Comment:
Our Url should be 
https://api.aspose.cloud/v3/imaging/...
Below code  removes trailing slash. 
Which Url do you expect to get  with
url = self.configuration.host + self.configuration.api_version  ... ?

Reply:
@s-zubov explained that he uses slash in another piece of code (I've already checked the working state of a branch with a build).